src/js/common/utils/helpers.js

```diff
@@ -101,14 +101,33 @@ export function dateToMoment(dateField) {
   });
 }
 
+export function dropTimeAndTimezone(date) {
+  let newDate = moment(date).format();
+  const indexOfTime = newDate.indexOf('T');
+  if (indexOfTime >= 0) {
+    newDate = newDate.substring(0, indexOfTime);
+  }
+  return newDate;
+}
+
 export function formatDateLong(date) {
   return moment(date).format('MMMM DD, YYYY');
 }
 
+export function formatDateWithoutTimeLong(date) {
```

Comment:
`formatDateWithoutTimeLong` & `formatDateWithoutTimeShort` seem excessive to me.  Just leave it up to the caller to make two calls like so,

```javascript 
var myDateTime = "1995-11-12T06:00:00.000+0000";
var date = dropTimeAndTimezone(myDateTime);
var formattedDate = formatDateLong(date);
```